File: 205. isIsomorphic.java
class Solution {
    public boolean isIsomorphic(String s, String t) {
        
        int[] sT = new int[127];
        
        Arrays.fill(sT,-1);
        
        int[] tT=new int[127];
        
        Arrays.fill(tT,-1);
        
        int i = 0 ;
        
        while(i<s.length()){
            
            if(sT[s.charAt(i)] != tT[t.charAt(i)]) return false;
            sT[s.charAt(i)]=i+1;
            tT[t.charAt(i)] = i+1;
            i++;
        }
        
        return true;
    }
}

File: 205. isIsomorphic.cpp
class Solution {
public:
    bool isIsomorphic(string s, string t) {
        int size = s.size();
        int sT[128] = {0},tT[128] = {0};
        for(int i = 0 ; i < size;i++){
            if(sT[s[i]] != tT[t[i]])
                return false;
            sT[s[i]] = i + 1;
            tT[t[i]] = i + 1;
        }
        
        return true;
    }
};
